modules/sno-clusters-reboot-without-drain.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes/nodes-nodes-working.adoc
+
+:_content-type: CONCEPT
+[id="sno-clusters-reboot-without-drain_{context}"]
+= Handling errors in {sno} clusters when the node reboots without draining application pods
+
+In {sno} clusters and in {product-title} clusters generally, a situation can arise where a node reboot occurs without first draining the node. This can occur where an application pod requesting devices fails with the `UnexpectedAdmissionError` error. `Deployment`, `ReplicaSet`, or `DaemonSet` errors are reported because the application pods that require devices start before the pod serving those devices and you cannot control the order of pod restarts.
+
+While this behavior is expected, it can lead to a pod remaining on the cluster even when it has failed to deploy successfully. The pod continues to report `UnexpectedAdmissionError`. This issue is mitigated by the fact that application pods are typically included in a `Deployment`, `ReplicaSet`, or `DaemonSet`. Having a pod in this error state is of little concern because another instance should be running. Being part of a `Deployment`, `ReplicaSet`, or `DaemonSet` guarantees the successful creation and execution of subsequent pods and ensures the successful deployment of the application.
+
+There is ongoing work upstream to ensure that such pods are gracefully terminated. Until that work is resolved, run the following command for a {sno} cluster to remove the failed pods:
+
+[source,terminal]
```

Comment:
```suggestion
[source,terminal,subs="+quotes"]
```

Reply:
Why is subs="+quotes" required?

Reply:
@agantony  I am not sure if this comes from a guideline. May be I am missing something here. If it does, you can quote it.

Reply:
Oh nevermind I see. thanks